app/src/main/java/de/xikolo/storages/base/SharedPreferenceLiveData.kt

```diff
@@ -0,0 +1,40 @@
+package de.xikolo.storages.base
+
+import android.content.SharedPreferences
+import androidx.lifecycle.LiveData
+
+abstract class SharedPreferenceLiveData<T>(var sharedPrefs: SharedPreferences,
```

Comment:
This is copy pasta from SO? Then please leave a comment with a link to the source here.